beep/tests/test_collate.py

```diff
@@ -11,12 +11,12 @@
 TEST_DIR = os.path.dirname(__file__)
 TEST_FILE_DIR = os.path.join(TEST_DIR, "test_files")
 
-
 class CollateTest(unittest.TestCase):
     def test_get_parameters_fastcharge(self):
         # Force a weird filename to be parsed
         source_directory = os.path.join(TEST_FILE_DIR)
-        date, chno, strname, protocol = get_parameters_fastcharge('2019-03-06-per_CH1234', source_directory)
+        with self.assertWarns(Warning):
```

Comment:
Cool! So I understand, this will pass if a warning is triggered, and error out in case of exception.

Reply:
Yes, and doesn't print the warning to screen since it expects it